.travis.yml

```
@@ -2,7 +2,19 @@ language: node_js
 
 node_js:
   - node
-  - 10
+  - 12
+  - 14
```

Slight OCD. Lower to higher versions. 

@samirsilwal Can you also change `"node": ">= 10.16.0"` in package.json to `"node": ">= 12.13.0"`

```suggestion
  - 14
  - 12
```